Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDocumentLayout.mm

```diff
@@ -66,26 +68,48 @@
 void PDFDocumentLayout::updateGeometry()
 {
     auto pageCount = this->pageCount();
-    m_pageBounds.clear();
+    m_pageGeometry.clear();
     m_documentBounds = { };
 
+    auto normalizeRotation = [](int degrees) {
+        if (degrees < 0)
+            degrees += 360 * (1 + (degrees / -360));
+
+        // Round to nearest 90 degree angle
+        degrees = std::round(static_cast<double>(degrees) / 90.0) * 90.0;
+
+        // Normalize in positive space
+        return degrees % 360;
+    };
+
+    auto normalizePageBounds = [&](FloatRect cropBox, int degrees) {
```

Comment:
is the `&` capture needed?

Reply:
Yes, for the lambda above.

Reply:
`normalizePageBounds` doesn't seem to use `normalizeRotation` though?